enhancements/two-nodes-openshift/2no.md

```diff
@@ -123,6 +127,26 @@ This functionality exists within RHEL-HA, but a wrapper will be provided to take
 
 When starting etcd, the OCF script will use etcd's cluster ID and version counter to determine whether the existing data directory can be reused, or must be erased before joining an active peer.
 
+
+### Summary of Changes
+
+At a glance, here are the components we are proposing to change:
+| Component                                                         | Change                                                                                                                                |
+| ----------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| [Feature Gates](#feature-gate-changes)                            | Add a new `DualReplicaTopology` feature to `TechPreviewNoUpgrade`                                                                     |
+| [Infrastructure API](#infrastructure-api-changes)                 | Add `DualReplica` as a new value for `ControlPlaneTopology`                                                                           |
+| [ETCD Operator](#etcd-operator-changes)                           | Add mode for external management of etcd, update scaling strategies to support 2 nodes, and add controller for initializing pacemaker |
```

Comment:
"external" etcd might be confusing, AFAIK it refers to etcd running completely outside the cluster. We just want to take over one of the multiple containers being managed by the etcd operator 🤔 
Maybe "Add mode for disabling the etcd container" is a better fit?

Reply:
I like the phrasing on that! Thanks!